pkg/api/v1/types.go

```diff
@@ -361,6 +361,8 @@ type VirtualMachineInstanceMigrationState struct {
 	Completed bool `json:"completed,omitempty"`
 	// Indicates that the migration failed
 	Failed bool `json:"failed,omitempty"`
+	// Indicates that the migration has been aborted
+	Aborted bool `json:"aborted,omitempty"`
```

Comment:
I'm not sure about the naming here.

"aborted" makes it sound the migration is in a finalized completed state when this boolean is really only requesting that an active migration be aborted.

would something like "cancelRequested" make more sense?

Reply:
yeah, done.